sql/_22_news_service_mysql_compatibility/_04_regular_expression/answers/_01_basic_test_05.answer

```diff
@@ -1,44 +1,44 @@
 ===================================================
-('a word file' regexp '[[:<:]]Word[[:>:]]')    
+('a word file' regexp '\bWord\b')    
 1     
 
 ===================================================
-('__pa word _file' regexp '[[:<:]]Word[[:>:]]')    
+('__pa word _file' regexp '\bWord\b')    
 1     
 
 ===================================================
-('__pa  word  _file' regexp '[[:<:]]Word[[:>:]]')    
+('__pa  word  _file' regexp '\bWord\b')    
 1     
 
 ===================================================
-('__pa  word _file' regexp '[[:<:]]Word[[:>:]]')    
+('__pa  word _file' regexp '\bWord\b')    
 1     
 
 ===================================================
-('__pa wrd  _file' regexp '[[:<:]]Wo?rd[[:>:]]')    
+('__pa wrd  _file' regexp '\bWo?rd\b')    
 1     
 
 ===================================================
-('__pa 7_1 _file' regexp '[[:<:]][^_a-z09]_[0-1][[:>:]]')    
+('__pa 7_1 _file' regexp '\b[^_a-z09]_[0-1]\b')    
 1     
 
 ===================================================
-('a word file' regexp binary '[[:<:]]Word[[:>:]]')    
+('a word file' regexp binary '\bWord\b')    
 0     
 
 ===================================================
-('__pa wordddd _file' regexp '[[:<:]]Word{2,3}[[:>:]]')    
+('__pa wordddd _file' regexp '\bWord{2,3}\b')    
 0     
 
 ===================================================
-('__pa a _file' regexp '[[:<:]][^a-z][[:>:]]')    
-0     
+('__pa a _file' regexp '\b[^a-z]\b')    
+1     
```

Comment:
this case seems to be divided.
> \b[^a-z] , [^a-z]\b

Reply:
I found that [[:<:]] and [[:>:]] does not have the same meaning with \b. I'll leave previous test cases that test the previous POSIX-style word boundary. and I'll add equivalent tests using \b.